src/components/Charts/mutationsTree.js

```diff
@@ -8,33 +8,22 @@ class MutationsTree extends React.Component {
         this.state = {};
     }
 
-    createVariants(variants, level){
-        return variants.flatMap((variant) => [
+    createVariants = (variants, level) => {
+        return (variants.flatMap((variant) => [
             `${'---'.repeat(level)}variant: ${variant.name} (${variant.mutations.map(mutation=>mutation.from+mutation.position+mutation.to).join(", ")})`,
             ...this.createVariants(variant.subs, level+1)
-         ])
-    }
-
-
-    componentDidMount() {
-        let variantsTree = this.createVariants(this.props.data, 0).map(i => {
+         ])).map(i => {
             return <p>{i}</p>
```

Comment:
isn't react complaining that array elements don't have any `key`?

Reply:
it doesn't, as far as I can see

Reply:
Are you sure? Doesn't it output warnings to the console in the runtime? :thinking:

Reply:
no, I see only the log outputs which were there in original rampart too.